.gitlab/ci/global.yml

```
@@ -68,3 +68,156 @@
   extends:
     - .default-cache
     - .default-before_script
+
+.notify-slack:
+  after_script:
+    - echo "=====Notify Slack====="
+    - |
+      if [ -n "${SLACK_ONLY_IF}" ]; then
+        ./Tests/scripts/slack_notifier.sh "$SLACK_TEST_TYPE" "$SLACK_ENV_RESULTS" "$SLACK_CHANNEL"
+      else
+        echo "Skipping Slack"
+      fi
```

Since I have a feeling we may want to include more steps in the `after_script` very soon, let's generalize this hidden job name.
```suggestion
.default-after-script:
  after_script:
    - section_start "Notify Slack"
    - |
      if [ -n "${SLACK_ONLY_IF}" ]; then
        ./Tests/scripts/slack_notifier.sh "$SLACK_TEST_TYPE" "$SLACK_ENV_RESULTS" "$SLACK_CHANNEL"
      else
        echo "Skipping Slack"
      fi
    - section_end "Notify Slack"
```